src/extensions/sessionrecording.ts

```diff
@@ -390,7 +375,13 @@ export class SessionRecording {
             return
         }
 
-        const { event, size } = ensureMaxMessageSize(truncateLargeConsoleLogs(rawEvent))
+        const throttledEvent = this.rateLimiter!.throttleMutations(rawEvent)
+
+        if (!throttledEvent) {
```

Comment:
this took me a second...

this is really `unthrottledEvent` right?

Reply:
Not really. Its an event with any content that it is throttled removed. It is then empty if everything was throttled to the point where there is nothing to do.

Reply:
Ah, ofc, the pedant in me says  then `eventWithThrottledMutations` instead but I like long variable names 🤣